src/server/url_shortening/routes/create_routes.js

```diff
@@ -0,0 +1,50 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { handleShortUrlError } from './lib/short_url_error';
+import { shortUrlAssertValid } from './lib/short_url_assert_valid';
+import { shortUrlLookupProvider } from './lib/short_url_lookup';
+import { createGotoRoute } from './goto';
+import { createShortenUrlRoute } from './shorten_url';
+
+
+export function createRoutes(server, config) {
+  const shortUrlLookup = shortUrlLookupProvider(server);
+
+  server.route(createGotoRoute({ server, config, handleShortUrlError, shortUrlAssertValid, shortUrlLookup }));
+  server.route(createShortenUrlRoute({ handleShortUrlError, shortUrlAssertValid, shortUrlLookup }));
```

Comment:
Why pass `shortUrlAssertValid` in as a parameter rather than import directly from those files?  Same with `handleShortUrlError`.

Reply:
no real reason. I was passing in `shortUrlLookup` so just figured to pass in everything to stay consistent.

Reply:
Do you ever see these functions needing to handle different implementations of those two parameters?  If not, I think it'd be more consistent and less complex if you imported and called directly.  The fewer function parameters you can get away with, the simpler the code (and easier to test).